tests/test_security_stop_report.py

```
@@ -0,0 +1,69 @@
+import asyncio
+import tempfile
+import shutil
+from pathlib import Path
+from typing import Any, Dict
```

Ruff (0.15.0)

[warning] `typing.Dict` is deprecated, use `dict` instead

(UP035)